frames/nikcircle.go

```diff
@@ -0,0 +1,17 @@
+package frames
+
+// This is the value stored in the FrameMap
+var Nikcircle = DefaultFrameType(Nikcircleframes)
+
+var Nikcircleframes = []string{
+       @@@@@@      
```

Comment:
This has a missing backtick (<code>`</code>) and won't compile.

I recommend adding a few more frames as well, as this won't actually animate with one frame.